.github/workflows/lint_test_build.yml

```diff
@@ -84,6 +84,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: ./.github/workflows/rafiki/env-setup
+      - run: pnpm --filter mock-account-servicing-entity build:deps
```

Comment:
Yes - a typescript error in `mock-account-servicing-lib` surfaces in this command. We are already doing `pnpm --filter mock-account-servicing-entity build` 2 steps down. I guess we can just switch the `typecheck` and `build` command instead of adding this new step.

For context, I did this after I saw CI fail the typecheck command because the `mock-account-servicing-lib` wasnt built. But bulding `mock-account-servicing-entity` first covers that.